lib/create.js

```diff
@@ -200,10 +200,12 @@ module.exports.deleteInvalidTemplate = function(options) {
  *     - `e` {Error} is null unless there was an error.
  */
 
-module.exports.downloadTemplate = function(options, callback) {
-    var destinationPath = module.exports.templatePath(options),
-        url = 'https://github.com/phonegap/phonegap-app-hello-world/archive/master.tar.gz';
-
+module.exports.downloadTemplate = function(options, callback, url) {
+    var destinationPath = module.exports.templatePath(options);
+	if (url === undefined) {
+        var url = 'https://github.com/phonegap/phonegap-app-hello-world/archive/master.tar.gz';
+    }
+    
```

Comment:
What's the purpose of this new addition? As far as I can tell, `url` will always be undefined.

Also, the function pattern should be `(options, callback)` where callback is always the last argument. If `url` is needed, then it should be `options.url`.